projects/laji/src/app/shared/navbar/navbar.component.html

```diff
@@ -24,7 +24,7 @@
           </div>
         </div>
         <div id="navbar" class="collapse navbar-collapse" [class.in]="openMenu">
-          <ul class="nav navbar-nav">
+          <ul class="nav navbar-nav" *ngIf="navbarService.navbarVisible">
```

Comment:
use `navbarService.navbarVisible$ | async` here directly and get rid of the `cdr.markForCheck()`